gravitee-apim-console-webui/src/management/api/documentation-v4/dialog/documentation-newt-ai-dialog/api-documentation-v4-newt-ai-dialog.component.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, Inject } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { FormBuilder, FormGroup } from '@angular/forms';
+
+import { ApiSpecGenState } from '../../../../../services-ngx/api-spec-gen.service';
+
+export interface ApiDocumentationV4NewtAiDialogData {
+  state: ApiSpecGenState;
+}
+
+@Component({
+  selector: 'api-documentation-v4-newt-ai-dialog',
+  templateUrl: './api-documentation-v4-newt-ai-dialog.component.html',
+  styleUrls: ['./api-documentation-v4-newt-ai-dialog.component.scss'],
+})
+export class ApiDocumentationV4NewtAiDialogComponent {
+  form: FormGroup;
```

Comment:
is this form necessary? it seems not but maybe a miss something

Reply:
Since I used a form as the structure. I decided to stick with it.
We may add more functionality in the future and the structure would be already there.